index.js

```diff
@@ -23,7 +27,7 @@ mongoose.connect(
   }
 );
 
-const port = 3000;
+const port = process.env.PORT || 3000;
 
```

Comment:
move this up... closer to where the connection string is defined. 

you might even do it in one line as 

`const{ CONNECTIONS_STRING, PORT=3000 } = process.env`

the above says "extract `CONNECTIONS_STRING` and `PORT` keys from `process.env` and use `3000` where `PORT` doesn't exist.

Reply:
done